jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -825,6 +825,8 @@ public boolean sendResponse(MetaData.Response info, ByteBuffer content, boolean
                 info = _response.newResponseMetaData();
             commit(info);
 
+            _request.onResponseCommit();
```

Comment:
I would move this line down to be at 837 immediately after `_combinedListener.onResponseBegin(_request);` because it is essentially the same event, but with a dedicated callback rather than the expense of a dynamic listener.